content/docs/user-guide/gems/reference/aws/aws-gamelift/cpp-api.md

```diff
@@ -5,15 +5,15 @@ description: "Learn how to use the C++ API for AWS GameLift Gem in O3DE"
 toc: true
```

Comment:
In the previous lines, make the following changes:

>title: C++ API for the AWS GameLift Gem
>description: Learn how to use the C++ API for the AWS GameLift Gem in Open 3D Engine (O3DE).

* Add "the" before "AWS GameLift Gem"
* Remove the quotation marks from the description.